Compass129Sources/COMfile/C12COM9.ASC

```diff
@@ -133,10 +133,10 @@ meminst	ld	de,txt_inst
 	jr	nc,okidoki
 	ld	de,txt_notfree
 	call	print
-	ld	a,#ae	;poke nieuwe opcode: res 5,(ix+0)
-	ld	(m_res6),a	;dit dient om de free-routine weer
-	xor	a	;gewoon te maken (was veranderd)
-	ld	(i_skipsrcmem),a	;default memuse in de datfile
+	ld	a,#ae	;poke new opcode: res 5,(ix+0)
+	ld	(m_res6),a	;this serves to display the free-routine

```

Comment:
'weer gewoon te maken' -> literally: 'make regular / as usual again'. So, perhaps:
'this serves to revert the free-routine (as it had been changed)'